### DISCUSSION TITLE
.NET MAUI Push Notifications using Azure Notification Hub. Part 1. Setup Azure Notification Hub

### DISCUSSION THREAD

**Discussion by giscus** [VladislavAntonyuk/vladislavantonyuk.github.io#110] [Announcements]
# .NET MAUI Push Notifications using Azure Notification Hub. Part 1. Setup Azure Notification Hub

The article demonstrates how to add push notifications to .NET MAUI application using Azure Notification Hub.

https://vladislavantonyuk.github.io/articles/.NET-MAUI-Push-Notifications-using-Azure-Notification-Hub.-Part-1.-Setup-Azure-Notification-Hub/

**Comment by plettb** [comment]
This solution seems to rely on "Cloud Messaging API (Legacy)", which was discontinued 6/20/2024. Have you got something that works without that?

**Reply by VladislavAntonyuk** [reply]
https://github.com/VladislavAntonyuk/vladislavantonyuk.github.io/issues/109
